app/views/filters/index.html.erb

```diff
@@ -29,9 +30,12 @@
                   merge(:auth_object => filter.role, :authorizer => @roles_authorizer))) %>
           </td>
         <% end %>
-        <td><%= _(filter.resource_class.try(:humanize_class_name) || filter.resource_type || '(Miscellaneous)') %></td>
+        <td>
+          <%= _(filter.resource_class.try(:humanize_class_name) || filter.resource_type || '(Miscellaneous)') %>
+        </td>
         <td><%= filter.permissions.map(&:name).join(', ') %></td>
         <td><%= checked_icon filter.unlimited? %></td>
+        <td><%= checked_icon filter.override? %></td>
         <td>
           <%= content_tag('span', link_to_if_authorized(filter.search || _('none'),
```

Comment:
please open a separate issue, I think this would deserve broader discussion and refactoring in whole Foreman, we display 'N/A' values elsewhere too, e.g. fact values table